evalml/objectives/utils.py

```diff
@@ -1,12 +1,26 @@
-from texttable import Texttable
 
 from .objective_base import ObjectiveBase
 
+from evalml import objectives
 from evalml.exceptions import ObjectiveNotFoundError
 from evalml.problem_types import handle_problem_types
 from evalml.utils.gen_utils import _get_subclasses
 
 
+def get_non_core_objectives():
+    """Get non-core objective classes.
+
+    Non-core objectives are objectives that are domain-specific. Users typically need to configure these objectives
+    before using them in AutoMLSearch.
+
+    Returns:
+        List of Objectivebase classes
```

Comment:
nit-picking to the max but `ObjectiveBase` :p

Reply:
Done!